### PR TITLE
[C#] Code Snippet for Atlas Search Autocomplete doesn't work

### PR DESCRIPTION
**Pull request by rachel-mack**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-37267>
Staging - <https://docs-mongodbcom-staging.corp.mongodb.com/csharp/docsworker-xlarge/DOCSP-37267/fundamentals/atlas-search/#std-label-csharp-atlas-search>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/fundamentals/atlas-search.txt [hunk 1]
```diff
@@ -106,6 +106,10 @@ collection using the string "Gib" in the ``make`` field.
    :language: csharp
    :dedent:
 
+.. note::
```
Comment (rustagir) [inline source/fundamentals/atlas-search.txt:109]: S: add newline after directive

File: source/fundamentals/atlas-search.txt [hunk 2]
```diff
@@ -106,6 +106,10 @@ collection using the string "Gib" in the ``make`` field.
...
+   If the field you are searching has a search index, it must be included in your ``Autocomplete`` call:
+   ``Search.Autocomplete(g => g.Make, "Gib"), indexName: <make_search_index>)``
```
Comment (rustagir) [inline source/fundamentals/atlas-search.txt:111]: Q: Is this issue unique to the `Autocomplete()` Search method? Can you please check with the Pm to verify if all of the code examples used on this page need to include the search index name?

**Review by rustagir — CHANGES_REQUESTED** [review]
requesting changes because of a question

File: source/fundamentals/atlas-search.txt (hunk 2 above)
Comment (rachel-mack) [reply source/fundamentals/atlas-search.txt:112]: [According to the PM](https://jira.mongodb.org/browse/DOCSP-37267?focusedCommentId=6149483&focusedId=6149483&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-6149483), yes, this is specific to Autocomplete and he did test all the methods except the Geo ones.

Comment (rustagir) [reply source/fundamentals/atlas-search.txt:112]: After reading the code example, I feel like the actual code example used in this section should be updated as well. The note could clarify that compared to the other examples, where the `default` index is used in the absence of the index name, this example explicitly requires the index name.

File: source/fundamentals/atlas-search.txt [hunk 3]
```diff
@@ -106,6 +106,10 @@ collection using the string "Gib" in the ``make`` field.
    :language: csharp
    :dedent:
 
+.. note::
+   
+   If the field you are searching has a search index, it must be included in your ``Autocomplete`` call. If a search index does not exist, the default index is used.
```
Comment (rustagir) [inline source/fundamentals/atlas-search.txt:111]:
```suggestion
   If the field you are searching on is indexed by a search index, you must pass the index name to the
   ``Autocomplete`` call. If a search index does not exist, the search uses the default index.
```

**Review by rustagir — APPROVED** [review]
lgtm with a final suggestion

**Pull request merged**